README.md

```diff
@@ -1,70 +1,164 @@
 # veritable-ui
 
-UI for `Veritable` utilizing [TSOA](https://tsoa-community.github.io/docs/getting-started.html), [HTMX](https://htmx.org/) and [@kitajs/html](https://www.npmjs.com/package/@kitajs/html) for JSX templating.
+A user interface for `Veritable` that allows to manage connections across supply chain and perform queries in relation to the supply chain. Utilizing [TSOA](https://tsoa-community.github.io/docs/getting-started.html), [HTMX](https://htmx.org/) and [@kitajs/html](https://www.npmjs.com/package/@kitajs/html) for JSX templates.
 
-## Setup
+#### Table of Contents
 
-To install dependencies:
+- [Setup](#setup)
+- [Getting Started](#getting-started)
+- [Development Mode/Local dev](#development-mode)
+- [Environment Variables](#environment-variables)
+- [Testing](#testing)
```

Comment:
You should add the other H2 sections in here.

Reply:
that should be updated